.github/workflows/maven.yml

```diff
@@ -174,22 +174,41 @@ jobs:
           server-username: MAVEN_USERNAME
           server-password: MAVEN_PASSWORD
 
+      - name: Check whether GPG Signing is Enabled
+        if: ${{ matrix.os == 'ubuntu-latest' }}
+        id: gpg-check
+        run: |
```

Comment:
Just for some context here, we can't simply use an `if` check on the existence of the secret because you can't access `secrets` from inside an `if` conditional and other workarounds I saw (place the secret in an ENV var and then `if: ${{ env.VAR != '' }}` online leaked the secret when Action Debug Logging is enabled

If however we use the secret inside a `run` block then GitHub Actions applies secret masking so even if the passphrase is present you can't see its value in the debug output